changelog.d/717.bugfix

```diff
@@ -0,0 +1 @@
+Add information about GitHub App Installs in 'update' state on the oauth status page
```

Comment:
Nitpick: end the sentence with a period (and we can consider updating the CI changelog script to check for that).

Reply:
Done. And yeah we should!